### PR TITLE
UI + tests

### PR DESCRIPTION
**Pull request by mlnkw**
* minor UI change
* implemented usage of fake data in ~/tests/list

### CODE REVIEW

File: TCMSApp/src/client/app/tests/test-create.html [hunk 1]
```diff
@@ -188,14 +189,14 @@
               <button type="button" class="btn btn-danger"><i class="fa fa-times"></i> Cancel</button>
             </div>
 
-          </div> <!-- closing panel-body -->
+          <!-- </div> closing panel-body -->
```
Comment (ITsvetkoFF) [inline TCMSApp/src/client/app/tests/test-create.html]: if we don't need some code, let's delete it (not comment out) :)

File: TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js [hunk 2]
```diff
@@ -102,7 +102,7 @@
                     var testCase = {};
 
                     testCase._id = faker.random.uuid();
-                    testCase.testName = faker.lorem.sentence(2, 7);
+                    testCase.testName = faker.lorem.sentence(2, 3);
```
Comment (ITsvetkoFF) [inline TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js]: 2, 3 means that we will get the sentence from 2 to 3 words? I think that 2-7 is more reasonable

File: TCMSApp/src/client/app/tests/test-create.html (hunk 1 above)
Comment (mlnkw) [inline TCMSApp/src/client/app/tests/test-create.html]: As I can see, we should reconsider our UI with whole crew.
Therefore, I decided to leave some trace for return till Monday :)

File: TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js (hunk 2 above)
Comment (mlnkw) [inline TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js]: 2-7 for a name? Why we have a description then?

Comment (mlnkw) [inline TCMSApp/src/client/app/blocks/fakers/fake-tests.factory.js]: 2, 3 means we get from 2 to 5 (2+3) words in the name of the test case

**Pull request closed without merging**